### PR TITLE
Document creating SSH servers via IaC

### PR DESCRIPTION
**Pull request by hugoShaka**
This PR contains documentation about registering SSH agentless nodes via IaC (tctl, operator, terraform).

The feature should be available in 15.2.0.

### CODE REVIEW

**Comment by hugoShaka** [comment]
Note for reviewers: the guide has TODOs in place of many command outputs. I will run the full guide, and report the command output. In the meantime, you can still review the guide itself; it will not change. I'm doing this to get this approved and merged before the end of next week for the 15.2 release.

File: docs/pages/management/dynamic-resources/agentless-ssh-servers.mdx [hunk 1]
```diff
@@ -0,0 +1,260 @@
+---
+title: Registering Agentless OpenSSH Servers with IaC
+description: Use infrastructure-as-code tooling to register Ggentless OpenSSH servers in Teleport.
```
Comment (r0mant) [inline docs/pages/management/dynamic-resources/agentless-ssh-servers.mdx:3]:
```suggestion
description: Use infrastructure-as-code tooling to register Agentless OpenSSH servers in Teleport.
```

File: docs/pages/management/dynamic-resources/agentless-ssh-servers.mdx [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+want to register in Teleport. Those files are called manifests and their syntax
+will vary based on the IaC tooling you'll use.
+
+Those manifests are typically versioned in a shared revision system like git.
+This allows you to keep track of all changes, follow standard code review
+procedures before changing resources in Teleport, and quickly redeploy your
+Teleport instance if needed.
+
+<Tabs>
+<TabItem label="tctl">
```
Comment (r0mant) [inline docs/pages/management/dynamic-resources/agentless-ssh-servers.mdx:94]: For the tab order, can we prioritize Terraform, then Kube Operator, and then tctl?

File: docs/pages/management/dynamic-resources/agentless-ssh-servers.mdx [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+TODO fill
+```
+
+</TabItem>
+</Tabs>
+
+## Step 3/4. Validate users were created
+
+Now that the IaC tooling has run, we'll validate that the users were properly
+created and granted the correct roles.
```
Comment (r0mant) [inline docs/pages/management/dynamic-resources/agentless-ssh-servers.mdx:225]: Why is this referring to users and roles (both in title and in the sentence)?

**Review by r0mant — COMMENTED** [review]
Mostly lgtm with a few suggestions.

File: docs/pages/includes/uuidgen.mdx [hunk 4]
```diff
@@ -0,0 +1,34 @@
+To generate a new UUID:
+
+<Tabs>
+<TabItem label="Linux">
+  Use `uuidgen`. This executable is installed by default on most machines. Else
```
Comment (marcoandredinis) [inline docs/pages/includes/uuidgen.mdx:5]: If you want another way to generate an UUID:
```
$ cat /proc/sys/kernel/random/uuid
8b02c7bd-b673-4f26-8522-c718add8ca5d
```

File: docs/pages/management/dynamic-resources/agentless-ssh-servers.mdx [hunk 5]
```diff
@@ -0,0 +1,260 @@
...
+## Step 3/4. Apply all manifests
+
+<Tabs>
+<TabItem label="tctl">
+
+Declare the server with the following command:
+
+```code
+$ tctl create -f openssh-node-resource.yaml
+TODO fill me
```
Comment (marcoandredinis) [inline docs/pages/management/dynamic-resources/agentless-ssh-servers.mdx:178]: Reminder :point_up:

**Review by marcoandredinis — APPROVED** [review]

File: docs/pages/management/dynamic-resources/agentless-ssh-servers.mdx (hunk 2 above)
Comment (hugoShaka) [reply docs/pages/management/dynamic-resources/agentless-ssh-servers.mdx:94]: I think we started with tctl on the other pages so I'd keep the order like this for now and do a PR to reorder in the future. I think that tctl would also make sense as a first choice as this has the lowest entrance barrier compared o TF and the operator

**Review by r0mant — APPROVED** [review]

**Review by tigrato — APPROVED** [review]

**Pull request merged**